### ISSUE TITLE
Cannot login

### ISSUE THREAD

**Issue by peterzandbergen** [brianstrauch/spotify-cli#33]
Concerns spotify version 1.9.0

After what seems a successful login, I get an error when I try to execute a command like status. I tried it both in Windows and in WSL2. Same result.

Windows 10
I run spotify login, the browser opens on Spotify login page for Spotify CLI, I login and get a success page in the browser. The cli does not seem to be happy:
```
spotify login
Error: You are not logged in: Run "spotify login"
Usage:
  spotify login [flags]

Flags:
  -h, --help   help for login
```

Any suggestions?

**Comment by brianstrauch** [comment]
Thanks for letting me know, fixed in 1.9.1! (Run `spotify update`)

**Comment by peterzandbergen** [comment]
You fixed it. Thanks

Peter Zandbergen
Tel: +31 6 460 55 872

linkedIn: peterzandbergen <https://www.linkedin.com/in/peterzandbergen>
